bmv2/Makefile.am

```diff
@@ -147,6 +150,43 @@ if P4THRIFT
     AM_CPPFLAGS += -DP4THRIFT
 endif
 
+# Openflow plugin
+if COND_PLUGIN_OPENFLOW
+unused_for_now = \
+@P4_NAME@/plugin/of/src/pd_wrappers.c    \
+@P4_NAME@/plugin/of/src/ofpat_groups.c   \
+@P4_NAME@/plugin/of/src/ofpat_state.c    \
+@P4_NAME@/plugin/of/src/ofpat_pipeline.c \
+@P4_NAME@/plugin/of/inc/pd_wrappers.h    \
+@P4_NAME@/plugin/of/inc/ofpat_groups.h   \
+@P4_NAME@/plugin/of/src/ofpat_state.h    \
+@P4_NAME@/plugin/of/inc/ofpat_pipeline.h \
+@P4_NAME@/plugin/of/inc/openflow.h
+
+of_sources = \
```

Comment:
can we move the definition of `of_sources` and `of_includes` above the `pd_files.ts` target?